src/components/Card/Card.tsx

```diff
@@ -19,7 +19,7 @@ type CardProps = {
   title: string;
   duration: number;
   image?: ImageData;
-  seriesId?: string;
+  isSeries?: boolean;
```

Comment:
Would it be more scaleable if we added `contentType` as a prop? I made this change in my hackweek branch and ended up using it for 2 different contentTypes (though series is still using the old flows and props there obviously): https://github.com/jwplayer/exp-ott-web-app/blob/main/src/components/Card/Card.tsx#L75

Reply:
I thought that the the main goal of this param is to define whether an item is a 'series' one. And that is why passing `seriesId` looked not as obvious in this case. We can pass `contentType`, but we also need to check custom params (`getSeriesIdFromCustomParams` function). Should we also pass all of them in this case?

Reply:
Passed item object to the Card.